flake.nix

```diff
@@ -176,9 +176,11 @@
           export LIBSSH2_SYS_USE_PKG_CONFIG=1
 
           export RUSTFLAGS="-Zthreads=0"
-        '' + pkgs.lib.optionalString useMoldLinker ''
+        '' + (if pkgs.stdenv.isLinux then ''
```

Comment:
`useMoldLinker` is unused after this change; should either keep using it or remove its definition (and probably move the comment down here?)

Reply:
It's still used in `RUSTFLAGS` in the call to `buildRustPackage` actually, but I should probably just remove it.

Reply:
Ah, I missed that.